### PR TITLE
Add InstallDefender to com.microsoft.office.plist

### PR DESCRIPTION
**Pull request by nixtar**
Adds support for the new InstallDefender key as per https://learn.microsoft.com/en-au/deployoffice/mac/preferences-office#app-installation and https://macadmins.software/shim/.

### CODE REVIEW

File: Manifests/ManagedPreferencesApplications/com.microsoft.office.plist [hunk 1]
```diff
@@ -136,6 +136,7 @@
 				</array>
 				<key>Installation</key>
 				<array>
+					<string>InstallDefender</string>
```
Comment (kevinmcox) [inline Manifests/ManagedPreferencesApplications/com.microsoft.office.plist:139]: Can we alphabetize this please?

File: Manifests/ManagedPreferencesApplications/com.microsoft.office.plist [hunk 2]
```diff
@@ -451,6 +452,24 @@
 			<key>pfm_type</key>
 			<string>integer</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>16.24</string>
```
Comment (kevinmcox) [inline Manifests/ManagedPreferencesApplications/com.microsoft.office.plist:457]: `pfm_app_min` should be `16.75` right?

**Review by kevinmcox — CHANGES_REQUESTED** [review]
I added some inline comments.

File: Manifests/ManagedPreferencesApplications/com.microsoft.office.plist (hunk 1 above)
Comment (nixtar) [reply Manifests/ManagedPreferencesApplications/com.microsoft.office.plist]: Done 👍

File: Manifests/ManagedPreferencesApplications/com.microsoft.office.plist (hunk 2 above)
Comment (nixtar) [reply Manifests/ManagedPreferencesApplications/com.microsoft.office.plist]: Yes of course, thanks for catching that.

**Comment by relgit** [comment]
Thanks @nixtar!

**Pull request merged**